tests/fixtures/links-schema-cache.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
Is this file still necessary since the original source is referenced directly in the schema: https://github.com/DACCS-Climate/DACCS-node-registry/pull/9/files#diff-2c4b9d4938eba1be2882c99ad66879ed9c28d8815f429b5aece574ddb6efad79R76-R77 ?

Reply:
I saw the offline test after, so I guess its fine. 
Does it actually work? That schema uses `"$dynamicRef": "https://json-schema.org/draft/2020-12/hyper-schema#meta"`, so it should still attempt an online resolution anyway?

Reply:
Yes, it's a bad idea to make your tests dependent on an internet connection. See here:

https://github.com/DACCS-Climate/DACCS-node-registry/pull/9/files#diff-aa57560853f73b4713b47a1ae27f28c76a227216db2eb068e32649322876ae94R45-R47

Reply:
Yes it actually works. Dynamic-refs are only resolved if the property is actually present. Since we're currently only testing links with rel, type, and href; the dynamic-ref is never triggered. 

If we ever change the tests to include some of those additional properties then, yes, we'd have to mock those ones as well.